app/package.json

```diff
@@ -12,7 +12,6 @@
 		"@types/react": "^17.0.0",
 		"@types/react-dom": "^17.0.0",
 		"genesys-dev-icons": "^0.1.12",
-		"genesys-react-components": "^0.1.0-alpha.15",
```

Comment:
make sure you run `yarn unlink` to add this back after a new version has been published.